web3.js/test/publickey.test.ts

```diff
@@ -38,6 +38,14 @@ describe('PublicKey', function () {
     expect(() => {
       new PublicKey('12345');
     }).to.throw();
+
+    expect(() => {
+      const rawJSONPublicKeyData = {
+        _bn: '135693854574979916511997248057056142015550763280047535983739356259273198796800000',
+      };
```

Comment:
Interesting. Where did you encounter such a legacy pubkey? Should we actually consider _supporting_ these – ie. parsing them if we discover something with this shape:

```
type LegacySerializedPublicKey = {
  _bn: string;
};
```